src/development/vortex/development/core/engine/trainer/lr_scheduler/__init__.py

```diff
@@ -1,4 +1,19 @@
 from torch.optim.lr_scheduler import *
-from .cosine_lr import *
-from .tanh_lr import *
-from .step_lr_with_burnin import *
\ No newline at end of file
+from .cosine_lr_with_warmup import *
+from .tanh_lr_with_warmup import *
+from .step_lr_with_warmup import *
+
+step_update_map = {
+    'batch_update':['CyclicLR',
+                    'OneCycleLR',
+                    ],
+    'epoch_update':['StepLRWithWarmUp',
+                    'CosineLRWithWarmUp',
+                    'TanhLRWithWarmUp',
+                    'StepLR',
+                    'MultiStepLR',
+                    'ExponentialLR',
+                    'CosineAnnealingLR',
+                    'CosineAnnealingWarmRestarts'
+                    ]
+}
```

Comment:
I prefer this kind of formatting, as it is more ordered indentation. But it's just my preference.
```suggestion
step_update_map = {
    'batch_update': [
        'CyclicLR',
        'OneCycleLR',
    ],
    'epoch_update': [
        'StepLRWithWarmUp',
        'CosineLRWithWarmUp',
        'TanhLRWithWarmUp',
        'StepLR',
        'MultiStepLR',
        'ExponentialLR',
        'CosineAnnealingLR',
        'CosineAnnealingWarmRestarts'
    ]
}
```